kalibro_client/base.py

```diff
@@ -74,6 +76,14 @@ def update(self, **kwargs):
         self.updated_at = response_body['updated_at']
 
     @classmethod
+    def find(cls, id):
+        response = cls.request('{}'.format(id), method='get')
```

Comment:
What do you think about passing id through the `params` parameter instead of interpolating strings?

Otherwise I would suggest to implement the interpolation inside the `request` method.